src/phillinois_app/app.py

```diff
@@ -19,6 +19,7 @@
 
 __version__ = 'GITHUB_TAG'
 __git_hash__ = 'GITHUB_SHA'
+__deployed__ = 'GITHUB_DEPLOYED'
```

Comment:
I think this should also be `__build_time__` or `__built__`

Reply:
Done.